ui/src/app/pages/analysis/runtime-panel.spec.tsx

```diff
@@ -836,7 +834,7 @@ describe('RuntimePanel', () => {
     const wrapper = await component();
 
     const costEstimator = () => wrapper.find('[data-test-id="cost-estimator"]');
-    expect(costEstimator()).toBeTruthy();
+    expect(costEstimator().exists()).toBeTruthy();
```

Comment:
These tests were not actually checking anything.  `find()` is apparently truthy even when it doesn't find something.